test/Microsoft.IdentityModel.Tokens.Tests/Validation/LifetimeValidationResultTests.cs

```diff
@@ -0,0 +1,289 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Diagnostics;
+using Microsoft.IdentityModel.Logging;
+using Microsoft.IdentityModel.TestUtils;
+using Xunit;
+
+namespace Microsoft.IdentityModel.Tokens.Validation.Tests
+{
+    public class LifetimeValidationResultTests
+    {
+        [Theory, MemberData(nameof(ValidateLifetimeTestCases), DisableDiscoveryEnumeration = true)]
+        public void ValidateLifetime(ValidateLifetimeTheoryData theoryData)
+        {
+            CompareContext context = TestUtilities.WriteHeader($"{this}.LifetimeValidatorTests", theoryData);
+
+            LifetimeValidationResult lifetimeValidationResult = Validators.ValidateLifetime(
+                theoryData.NotBefore,
```

Comment:
I think we are missing a test case for null notbefore?

Reply:
Added test case.